lib/Viewer.js

```diff
@@ -153,8 +154,9 @@ inherits(Viewer, Diagram);
  *
  * @param {String} xml the BPMN 2.0 xml
  * @param {Function} [done] invoked with (err, warnings=[])
+ * @param {String} ID of the diagram to render (if null or non-existent, the first will be rendered)
  */
-Viewer.prototype.importXML = function(xml, done) {
+Viewer.prototype.importXML = function(xml, done, diagramId) {
```

Comment:
Can you please keep the current pattern of placing the callback at the end?
```suggestion
Viewer.prototype.importXML = function(xml, diagramId, done) {
```

Reply:
Are you sure is this needed?

The drawback to follow this pattern is that it would break all existing external code (e.g. camunda modeler) using this interface.

Reply:
No, you wouldn't, if `diagramId` is not present, `done` will be considered the second parameter.

Reply:
```
function check(first, optional, done) {
  if (isFunction(optional)) {
    done = optional;
    optional = null;
  }
  ...
}
```